pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -82,6 +83,58 @@ const (
 	imageDigestExporterContainerName = "step-image-digest-exporter"
 )
 
+type TaskRunWithTask struct {
+	taskRunSteps []v1alpha1.StepState
+	mapForSort map[string]int
+}
+
+func (trt *TaskRunWithTask) Init(taskRunSteps []v1alpha1.StepState, taskSpecSteps []corev1.Container) {
```

Comment:
nit: I feel we don't need this and we could just declare it *inline* [here](https://github.com/tektoncd/pipeline/pull/963/files#diff-b52b9f382ad878e6aa2a85921bb19065R401)

```go
trt := &TaskRunWithTask{
	taskRunSteps: taskRunSteps,
        mapForSort: trt.constructTaskStepsSorter(taskSpecSteps),
}
```

But it's more a aesthetic thing than anything, so I am ok with both I think :angel:

Reply:
Thx.